Automattic/wp-calypso
# Jetpack Onboarding: Add illustration to summary step

This PR adds a fireworks illustration to the Summary step. Fixes #23461.

Preview:

![](https://cldup.com/-qEG0j72zm.png)

To test:
* Start the flow here: http://yourgroovydomain.com/wp-admin/admin.php?page=jetpack&action=onboard&calypso_env=development
* Skip to the Summary step
* Verify the new illustration is there and looks good in all resolutions.
